pages/api/admin/users/readMany.ts

```diff
@@ -1,28 +1,29 @@
 import { PrismaClient } from "@prisma/client";
 import { NextApiRequest, NextApiResponse } from "next";
 import sanitizeUser from "utils/sanitizeUser";
+import { USER_PAGE_SIZE } from "../../../../constants";
 
 const prisma = new PrismaClient();
 
-export const USER_PAGE_SIZE = 21;
-
 export default async (
   req: NextApiRequest,
   res: NextApiResponse
 ): Promise<void> => {
   const pageNumber: number = Number(req.query.pageNumber) || 0;
   try {
     const user = await prisma.user.findMany({
-      skip: USER_PAGE_SIZE * pageNumber,
-      take: USER_PAGE_SIZE,
+      skip: Number(USER_PAGE_SIZE) * pageNumber,
+      take: Number(USER_PAGE_SIZE),
     });
 
+    const userCount = await prisma.user.count();
+
     if (!user) {
       res
         .status(404)
         .json({ statusCode: 404, message: "User does not exist." });
     } else {
-      res.json({ users: user.map(sanitizeUser) });
+      res.json({ users: user.map(sanitizeUser), total: userCount });
```

Comment:
Instead of responding with the `total` as a separate property, can we use `users.length` in the response?

Reply:
@ethanlee16 don't think I can use users.length because that would only give me the size of one backend page (21 users) because we only load that many at a time. Figured out that `await prisma.user.count();` would give me the total number of entries in the table